book.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  # attr_reader :rentals
```

Comment:
[OPTIONAL] Please remove commented code from the codebase because it makes your codebase messy if you want to check your previous code then check your GitHub commit history :slightly_smiling_face: